### PR TITLE
Update README.md files

### PR DESCRIPTION
**Pull request by superstas**
Add "Scheduling In Go - Part II" link to reading/README.md
Update links markup in topics/go/concurrency

### CODE REVIEW

File: topics/go/concurrency/channels/README.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ The behavior of a channel is directly influenced by its current `State`. The sta
 ## Links
 
 [Channel Communication](https://golang.org/ref/mem#tmp_7)  
-http://blog.golang.org/share-memory-by-communicating  
-http://www.goinggo.net/2014/02/the-nature-of-channels-in-go.html  
-[A Retrospective on SEDA](http://matt-welsh.blogspot.com/2010/07/retrospective-on-seda.html) - Matt Welsh  
+[Share Memory By Communicating](http://blog.golang.org/share-memory-by-communicating) - Andrew Gerrand
```
Comment (ardan-bkennedy) [inline topics/go/concurrency/channels/README.md]: All these lines need 2 extra spaces at the end so they CRLF.

File: topics/go/concurrency/data_race/README.md [hunk 2]
```diff
@@ -23,12 +23,11 @@ This content is provided by Scott Meyers from his talk in 2014 at Dive:
...
 ## Links
 
-[Eliminate False Sharing](http://www.drdobbs.com/parallel/eliminate-false-sharing/217500206)
-
+[Eliminate False Sharing](http://www.drdobbs.com/parallel/eliminate-false-sharing/217500206) - Herb Sutter  
 [The Go Memory Model](https://golang.org/ref/mem)  
-http://blog.golang.org/race-detector  
-http://www.goinggo.net/2013/09/detecting-race-conditions-with-go.html  
-https://golang.org/doc/articles/race_detector.html
+[Introducing the Go Race Detector](http://blog.golang.org/race-detector) - Dmitry Vyukov and Andrew Gerrand
```
Comment (ardan-bkennedy) [inline topics/go/concurrency/data_race/README.md]: All these lines need 2 extra spaces at the end so they CRLF.

File: topics/go/concurrency/goroutines/README.md [hunk 3]
```diff
@@ -26,18 +26,18 @@ Goroutines are functions that are created and scheduled to be run independently
 
 ## Links
 
-http://blog.golang.org/advanced-go-concurrency-patterns  
-http://blog.golang.org/context  
-http://blog.golang.org/concurrency-is-not-parallelism  
-http://talks.golang.org/2013/distsys.slide  
+[Advanced Go Concurrency Patterns](https://blog.golang.org/advanced-go-concurrency-patterns) - Sameer Ajmani   
+[Go Concurrency Patterns: Context](https://blog.golang.org/context) - Sameer Ajmani   
+[Concurrency is not parallelism](https://blog.golang.org/concurrency-is-not-parallelism) - Rob Pike
```
Comment (ardan-bkennedy) [inline topics/go/concurrency/goroutines/README.md]: All these lines need 2 extra spaces at the end so they CRLF.

Double check all links please.

Comment (superstas) [reply topics/go/concurrency/goroutines/README.md]: Sure, thanks for checking!
I've added four spaces to each link, PTAL.

**Pull request merged**